docs/code.rst

```diff
@@ -205,6 +205,14 @@ Operators
 .. autoclass:: airflow.contrib.operators.gcp_sql_operator.CloudSqlInstanceImportOperator
 .. autoclass:: airflow.contrib.operators.gcp_sql_operator.CloudSqlInstancePatchOperator
 .. autoclass:: airflow.contrib.operators.gcp_sql_operator.CloudSqlQueryOperator
+.. autoclass:: airflow.contrib.operators.gcp_vision_operator.CloudVisionProductSetCreateOperator
```

Comment:
Sort alphabetically by class name and then package.

Reply:
Done